cli.js

```diff
@@ -127,9 +128,14 @@ function setOptionDefaults(module) {
 function setCommandDefaults(module) {
   let self = {};
 
+  if (module.usage) {
```

Comment:
For the commands usages that use $0 since they don't have access to that variable in the other files like in `commands/inbound-domains.js`